lib/js-file.js

```diff
@@ -368,6 +404,32 @@ JsFile.prototype = {
         });
     },
     /**
+     * Returns comment node directly followed by a token
+     *
+     * @param {Object} token
+     * @returns {Object|undefined}
+     */
+    getCommentAfterToken: function(token) {
+        if (!this._tokenCommentsAfterIndex) {
```

Comment:
Is it fine if we checking it in methods? We can call here `_buildCommentIndex` and move check there. Also we can build that index every time just after token index building.